### DISCUSSION TITLE
Pinia stores getting cached in Vercel

### DISCUSSION THREAD

**Discussion by ejerskov** [vuejs/pinia#2831] [Help and Questions] [answered]
I have a basic implementation of pinia stores on my site and when running it on Vercel with caching (ISR) all stores are cached and then available for all users.
Some of the stores could make sense to cache for all but my authStore definitely shouldn't.

Is it default behaviour for the stores to be cached? And what can I do to avoid it?

**Comment by posva** [comment, marked as answer]
You might be using the stores wrong: https://pinia.vuejs.org/ssr/#Using-the-store-outside-of-setup-
